sql/migrations/jobsdb/000002_alter_dataset_tables.up.tmpl

```diff
@@ -0,0 +1,14 @@
+{{range .Datasets}}
+    ALTER TABLE {{$.Prefix}}_jobs_{{.}} ALTER COLUMN created_at set DEFAULT NOW();
+    ALTER TABLE {{$.Prefix}}_jobs_{{.}} ALTER COLUMN expire_at set DEFAULT NOW();
+    ALTER TABLE {{$.Prefix}}_jobs_{{.}} ADD COLUMN IF NOT EXISTS user_id TEXT NOT NULL DEFAULT '-1';
+
+    -- Convert job_state to varchar, drop type job_state_type
+    ALTER TABLE {{$.Prefix}}_job_status_{{.}} ALTER COLUMN job_state TYPE VARCHAR(255);
```

Comment:
Down for this is not written. It might not hurt, but leaving this comment just to check if this was intentional.

Reply:
I did not add because there is no workflow for reverting migrations, but I should add it for the sake of completeness.